subgraph-js/CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 This CHANGELOG pertains only to Apollo Federation packages in the 2.x range. The Federation v0.x equivalent for this package can be found [here](https://github.com/apollographql/federation/blob/version-0.x/subgraph-js/CHANGELOG.md) on the `version-0.x` branch of this repo.
 
 ## 2.3.0-beta.2
-- `@tag` directive support for the `SCHEMA` location [PR #2314](https://github.com/apollographql/federation/pull/2314).
+- `@tag` directive support for the `SCHEMA` location. This has been added to the 2.3 version of the federation spec, and you must declare usage of this spec version in your subgraph to use this functionality. [PR #2314](https://github.com/apollographql/federation/pull/2314).
```

Comment:
Can we be extra clear on the action item that's required for users? (you need to bump your `@link` federation spec version to v2.3 in your type definitions)

Providing a snippet certainly would remove any uncertainty.
```
@link(url: "https://specs.apollo.dev/federation/v2.3", import: [...])
```

Reply:
@trevor-scheer Makes sense, made it more explicit in the entry. (I also removed the `import` since it's not strictly needed to run into this issue.)